php-checks/src/main/java/org/sonar/php/checks/PHPDeprecatedFunctionUsageCheck.java

```diff
@@ -147,6 +148,15 @@ public void visitMemberAccess(MemberAccessTree tree) {
     super.visitMemberAccess(tree);
   }
 
+  @Override
+  public void visitFunctionDeclaration(FunctionDeclarationTree tree) {
+    NameIdentifierTree name = tree.name();
+    if (name.text().equalsIgnoreCase(ASSERT_FUNCTION)) {
+      context().newIssue(this, name, "Use instead the standard \"assert\" function.");
```

Comment:
`"Use the standard \"assert\" function instead of declaring a new assert function."`